config/manager/kustomization.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - deployment.yaml
+
+# Uncomment to enable HTTPS for the registry
+#patches:
+#- path: ./patches/add_root_certificates.yaml
```

Comment:
is it working without the patch? You already try to mount the secret in `deployment.yaml` above.

Reply:
I forgot to remove it. :D

Reply:
But it does work, yeah.

Reply:
Verified by running: `kustomize build ./config/default > ./install.yaml`.